eta/core/image.py

```diff
@@ -368,6 +372,51 @@ def merge_labels(self, image_labels):
         self.add_image_attributes(image_labels.attrs)
         self.add_objects(image_labels.objects)
 
+    @property
+    def has_schema(self):
```

Comment:
I didn't initially implement schemas at all for `ImageLabels` --- only at the `ImageSetLabels` level --- because it is just one image and the labels are pretty self-evident. But I can see why it is convenient to use them to filter.

So, please do one of the following:

(i) fully support the schema-related methods that `ImageSetLabels` and `VideoLabels` support. Namely `set_schema(), freeze_schema(), remove_schema(), get_schema(), get_active_schema()`. And add `schema` to `attributes()` and `from_dict()`, which are missing here

(ii) remove `schema` as an attribute and rename `set_schema_and_filter()` to `filter_by_schema()` to clarify that the schema won't be stored

Reply:
Oh I see. I was curious why videoLabels had it, but imageLabels did not.  I did not investigate imagesetlabels.  I'll look into this.

Reply:
I opt'ed for (ii) for the sake of not introducing more complexity in this PR.